Makefile

```diff
@@ -91,7 +91,7 @@ generate-in-docker: hack/jsonnet-docker-image
 	--rm \
 	-u=$(shell id -u $(USER)):$(shell id -g $(USER)) \
 	-v `pwd`:/go/src/github.com/coreos/prometheus-operator \
-	po-jsonnet make generate
+	po-jsonnet make $(MFLAGS) generate
```

Comment:
Maybe worth adding a small comment on what `$(MFLAGS)` does in this particular case.